### PR TITLE
Fixed a wrong sentence structure 

### PR DESCRIPTION
**Pull request by utsav0**
# Welcome to the JavaScript community

[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [ ] Add an algorithm?
- [x] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/FindLcm.js [hunk 1]
```diff
@@ -26,11 +26,7 @@ const findLcm = (num1, num2) => {
   let maxNum
   let lcm
   // Check to see whether num1 or num2 is larger.
-  if (num1 > num2) {
-    maxNum = num1
-  } else {
-    maxNum = num2
-  }
+  maxNum = Math.max(num1, num2)
```
Comment (raklaptudirm) [inline Maths/FindLcm.js:29]: Remove this change from here.

File: Bit-Manipulation/SetBit.js [hunk 2]
```diff
@@ -7,7 +7,7 @@
  * bit binary representations of the number and
  * returns a number after comparing each bit.
  *
- * 0 | 0 -> 0
+ * 0 | 0 -> 0
```
Comment (raklaptudirm) [inline Bit-Manipulation/SetBit.js:10]: Remove these changes from here.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

File: Maths/FindLcm.js [hunk 3]
```diff
@@ -26,16 +26,16 @@ const findLcm = (num1, num2) => {
...
   lcm = maxNum
 
   while (true) {
     if (lcm % num1 === 0 && lcm % num2 === 0) break
-    lcm += maxNum
+    if (num1 > num2) {
```
Comment (raklaptudirm) [inline Maths/FindLcm.js:34]: Why did you change this?

Comment (utsav0) [reply Maths/FindLcm.js]: It was by mistake. I was confused that's why.

File: Bit-Manipulation/SetBit.js [hunk 4]
```diff
@@ -20,7 +20,7 @@
  * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR
  */
 
-/**
+/*
```
Comment (raklaptudirm) [inline Bit-Manipulation/SetBit.js:23]: Also remove this change.
